package.json

```diff
@@ -202,7 +202,7 @@
     "postcss": "8.3.5",
     "postcss-loader": "4.1.0",
     "postcss-modules": "4.1.3",
-    "prettier": "2.2.1",
+    "prettier": "^2.2.1",
```

Comment:
dependencies here  should not use carets

Reply:
Yes, these version changes should be reverted, and instead add `prettier`, `workspace-tools` to the synpack config for bundle-size below.